packages/types/src/chain-adapters/ethereum.ts

```diff
@@ -46,7 +46,11 @@ export type QuoteFeeData = {
 }
 
 export type BuildTxInput = {
-  fee: string
+  gasPrice: string
   gasLimit: string
   erc20ContractAddress?: string
 }
+export type GetFeeDataInput = {
+  contractAddress?: string
+  from: string
```

Comment:
since both chains and presumably more in the future have some sense of a from address/xpub. I wonder if this would make more sense as part of the base GetFeeDataInput type. We can just use the `pubkey` vernacular to imply "this is your from address or xpub"

Reply:
ya thats a good point. Id prefer to leave it for now because everything is working as-is but its not hard to change and is a relevant point

Your comment also reminds me that a lot of this chainSpecific stuff seems kindof pointless